### PR TITLE
Deprecate public node, public wallet, Howto for wallet.parity.io

### PR DESCRIPTION
**Pull request by Tbaut**
Overall cleanup for wallet.parity.io and `--public-node` deprecation
- replaces the original pages
- removes unused images

### CODE REVIEW

**Comment by 5chdn** [comment]
Is anyone actually working on a replacement for wallet.parity.io ?

**Comment by Tbaut** [comment]
yes, it's @maciejhirsz

File: Public-Node.md [hunk 1]
```diff
@@ -2,22 +2,4 @@
...
-The most important difference to a regular (local) Parity wallet is the way your keys are stored. Instead of creating a local profile on your disk, **your keys are entirely kept in your browser's local storage!** Always  [backup](Backing-up-&-Restoring.md) your recovery phrase and export the private keys (JSON) before using the public node on any of your devices.
-
-List of additional features of a public node:
-
-- @TODO
-
-List of disabled features:
-
-- @TODO
+The flag `--public-node` has been deprecated and should not be used anymore. Parity Ethereum client does not support it anymore starting from v1.12.
```
Comment (5chdn) [inline Public-Node.md]: ... starting from v1.9 ...

**Comment by Tbaut** [comment]
updated, it's minimal now :)

**Comment by Tbaut** [comment]
@5chdn please have a look (and merge if you're ok) as the new wallet.parity.io is live

**Comment by 5chdn** [comment]
Perfect

**Pull request merged**